### PR TITLE
Fix FCI processing and split Ubuntu 22 and Ubuntu 24

### PR DESCRIPTION
**Pull request by murdaca**
- split Ubuntu 22 and Ubuntu 24
- fix config location
- bug fix for ubuntu 22 https://bugs.launchpad.net/ubuntu/+source/libpod/+bug/2024394 for podman network

### CODE REVIEW

File: fci-processing/fci-processing-tasks.yaml [hunk 1]
```diff
@@ -10,10 +10,19 @@
...
+
+- name: bug fix for ubuntu 22 https://bugs.launchpad.net/ubuntu/+source/libpod/+bug/2024394 part 2
+  shell: dpkg -i containernetworking-plugins_1.1.1+ds1-3build1_amd64.deb
+  when: ansible_distribution == 'Ubuntu' and ansible_distribution_version == '22.04'
+
 - name: Install the latest version of aardvark-dns
   package:
     name: aardvark-dns
     state: latest
+  when: ansible_distribution == 'Ubuntu' and ansible_distribution_version == '24.04'
```
Comment (mraspaud) [inline fci-processing/fci-processing-tasks.yaml:25]: shouldn’t this be `>= 24.04`?

Comment (mraspaud) [reply fci-processing/fci-processing-tasks.yaml:25]: don’t comparisons with strings work?

Comment (murdaca) [reply fci-processing/fci-processing-tasks.yaml]: I added >=, tested and works fine!

**Review by mraspaud — APPROVED** [review]
LGTM!

**Pull request merged**